pkg/gardenlet/controller/seed/lease_control.go

```diff
@@ -44,30 +45,26 @@ func (c *Controller) seedLeaseAdd(obj interface{}) {
 	c.seedLeaseQueue.Add(key)
 }
 
-const (
-	// LeaseDurationSeconds defines how long the lease is valid (used for Lease.spec.leaseDurationSeconds).
-	LeaseDurationSeconds = 2
-	// LeaseResyncSeconds defines how often (in seconds) the seed lease is renewed.
-	LeaseResyncSeconds = 2
-	// LeaseResyncGracePeriodSeconds is the grace period for how long the lease may not be resynced before the health status
-	// is changed to false.
-	LeaseResyncGracePeriodSeconds = LeaseResyncSeconds * 10
-)
+// LeaseResyncGracePeriod is the amount of missed lease resyncs before the health status
+// is changed to false.
+const LeaseResyncGracePeriod = 10
```

Comment:
Shouldn't this also be configurable? If I have a high lease duration then I want this probably to be smaller?

Reply:
This can be the use case to make this field configurable.

Reply:
I have made `LeaseResyncGracePeriodSeconds` configurable. But I am not sure whether we should write Unit test for this , as there is no test available for [gardenlet.go](https://github.com/gardener/gardener/blob/master/cmd/gardenlet/app/gardenlet.go) where `LeaseResyncGracePeriodSeconds` is being called.